docs/INSTALL.md

```diff
@@ -37,29 +47,87 @@ $ gcloud auth login
 $ gcloud auth application-default login
 ```
 
-### Custom terraform-dcos variables
+#### Supported Operating Systems
+
+Here is the [list of operating systems supported](https://github.com/dcos-terraform/terraform-template-gcp-tested-oses/tree/master/platform/cloud/gcp).
+
+#### Supported DC/OS Versions
+
+Here is the list of DC/OS versions supported on dcos-terraform natively:
 
-The default variables are tracked in the [variables.tf](../variables.tf) file. Since this file can be overwritten during updates when you may run `terraform get --update` when you want to fetch new releases of DC/OS to upgrade too, its best to use the cluster_profile.tfvars and set your custom terraform and DC/OS flags there. This way you can keep track of a single file that you can use manage the lifecycle of your cluster.
+- [OSS Versions](https://github.com/dcos-terraform/terraform-template-dcos-core/tree/master/open/dcos-versions)
+- [Enterprise Versions](https://github.com/dcos-terraform/terraform-template-dcos-core/tree/master/ee/dcos-versions)
 
-To apply the configuration file, you can use this command below.
+**Note**: Master DC/OS version is not meant for production use. It is only for CI/CD testing.
 
-### `cluster_profile.tfvars`
+To apply the configuration file and you accept all the default variables you can use this command below, otherwise you can continue along with this guide.
 
 ```bash
-# or similar depending on your environment
-echo 'public_ssh_key_path = "~/.ssh/id_rsa.pub"' >> cluster_profile.tfvars
-# lets set the clustername
-echo 'cluster_name = "my-ee-cluster"' >> cluster_profile.tfvars
-# we at mesosphere have to tag our instances with an owner and an expire date.
-echo 'tags = ["prod", "staging", "kubernetes"]' >> cluster_profile.tfvars
-# we have to explicitly set the version.
-echo 'dcos_version = "1.10.8"' >> cluster_profile.tfvars
-# we can set the azure location
-echo 'region = "us-west1"' >> cluster_profile.tfvars
-# set the google project id
-echo 'project_id = "massive-bliss-781"' >> cluster_profile.tfvars
+terraform apply
+```
+
+## Advanced YAML Configuration
+
+We have designed this project to be flexible. Here are the example working variables that allows very deep customization by using a single `main.tf` file.
```

Comment:
Rewrite as:
The configuration templates using Terraform are designed to be flexible. Here is an example of working variables that allows deep customization by using a single main.tf file.